Bootstrap V4

rdvelazquez opened the pull request:
__Contents of PR__:
 - Fixes the navbar (load button works again; buttons floated to the right)
 - Positions the scroll-spy and makes it only display on lg or above
 - Fixes the headers for the tables/graphs
 - Replaces glyph icons with font-awesome icons in methods files
 - Fixes Y-axis toggling for FEL site plot graph
 - Fixes scrollSpy (thank you @stephenshank! )
 - Multiple other fixes

stephenshank commented:
aBSREL
- Branch tooltip in tree component does not display properly (we can make this an issue instead of fixing here)
- In "Detailed results" table, the chart modal appears to be broken

BUSTED
- Most tooltips and popovers do not appear to be working
- The model fits chart modal appears to be broken

RELAX
- Bar plots are no longer horizontally centered

FEL
- Site plot should have export buttons pulled to the right
- Tree component navbar does not fit (settings button is moved to a new line)

MEME
- p-value threshold in summary and rows in MEME table are not in sync (though this is apparently also the case on vision.hyphy.org... can make this an issue)

SLAC
- Popovers do not appear to work

GARD
- Popovers do not appear to work

DatamonkeyTable
- Export CSV button should have a disk icon

stephenshank commented on src/jsx/busted.jsx:
```diff
@@ -396,7 +396,7 @@ var BUSTEDSiteChartAndTable = React.createClass({
             <div className={"form-group" + (this.state.CERwarning ? " has-error" : "")}>
               <label for="er-constrained-threshold">
-                Constrained Test Statistic
+                Constrained Test Statistic TEST
```
Why is the word `TEST` here?

rdvelazquez replied:
This was a remnant of debugging and has been removed. Nice catch.

rdvelazquez commented:
Thank you for the review and testing! I've addressed all your comments.

> aBSREL -Branch tooltip in tree component does not display properly (we can make this an issue instead of fixing here)

I don't think this is related to the bootstrap upgrade but it is covered by #133.
